### PR TITLE
Content expansion

### PR DESCRIPTION
**Pull request by andyjih**
I started working on `/docs` and `/blog` pages, but decided to just prioritize main site content in this pass.

### CODE REVIEW

File: pages/docs/support/permissions.mdx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
 title: "Permissions"
-date: "2021-02-01"
+date: "2021-07-20"
 pullQuote: "Grouparoo has a robust permission system you can use to limit access to your employees and applications."
 ---
 
 Grouparoo has a robust permission system you can use to limit access to your employees and applications. The Grouparoo permission model operates on Teams (for individual users) and API Keys (for applications). Consider making multiple Teams and API Keys for each use case.
 
+The first team you create will have full read and write access to all features and capabilities. After you make your first team, you can control which Teams have read and/or write access to different features and capabilities. Controling permissions at the Team level make it easy for different groups of people in your organization to use Grouparoo. For example, an engineer will want access to reading and writing all types of models, while someone on the revenue ops or marketing ops team will likely only want write access to Destinations and Groups.
```
Comment (teallarson) [inline pages/docs/support/permissions.mdx:9]: This is a really helpful explanation of teams!  Glad you added it.  Do we want to mention that these permissions are only relevant to the UI?  Anyone could go in and directly edit the json files.

File: pages/integrations/bigquery-to-salesforce.tsx [hunk 2]
```diff
@@ -234,6 +235,79 @@ export default function BigQueryToSalesforce() {
...
+        id="about-bigquery-and-salesforce"
+        className="integrationsPageSection"
+      >
+        <Container>
+          <Row>
+            <Col>
+              <h2>About Google Cloud BigQuery</h2>
+              <p>
+                BigQuery is one of Google's main products in their Google Cloud
+                Platform. BigQuery is serverless and scalable data warehouse
```
Comment (teallarson) [inline pages/integrations/bigquery-to-salesforce.tsx]: is a serverless and scalable data warehouse?  is serverless and scalable data warehouse infrastructure?

File: pages/integrations/bigquery-to-salesforce.tsx [hunk 3]
```diff
@@ -234,6 +235,79 @@ export default function BigQueryToSalesforce() {
...
+              <p>
+                Salesforce is a leader in the SaaS-based CRM space. They have an
+                extensive product suite that encompasses marketing, sales,
+                support, and many other functions. Salesforce allows companies
+                and organizations to manage the customer experience and journey
+                across many different channels.
+              </p>
+              <small>
+                <Link href="/integrations/destinations/salesforce">
+                  <a>Learn more about Grouparoo's Salesforce plugin</a>
```
Comment (teallarson) [inline pages/integrations/bigquery-to-salesforce.tsx]: Salesforce data integration?

File: pages/integrations/postgres-to-hubspot.tsx [hunk 4]
```diff
@@ -223,6 +225,76 @@ export default function PostgresToHubspot() {
...
+              <p>
+                Hubspot is a powerful marketing, sales, and customer support
+                software platform. Hubspot was started in 2006 and has since
+                grown to become one of the leaders in the SaaS CRM space. They
+                help companies of all sizes communicate with their customers and
+                manage their customer relationships.
+              </p>
+              <small>
+                <Link href="/integrations/destinations/hubspot">
+                  <a>Learn more about Grouparoo's Hubspot plugin</a>
```
Comment (teallarson) [inline pages/integrations/postgres-to-hubspot.tsx]: Hubspot integration?

File: pages/integrations/postgres-to-salesforce.tsx [hunk 5]
```diff
@@ -231,6 +232,79 @@ export default function PostgresToSalesforce() {
...
+              <h2>About PostgreSQL</h2>
+              <p>
+                PostgreSQL (or Postgres for short) is a powerful, open source
+                object-relational database. Postgres is one of the most popular
+                database types in modern application development, and it's also
+                the foundation for Amazon's Redshift data warehouse product.{" "}
+              </p>
+              <small>
+                <Link href="/integrations/sources/postgres">
+                  <a>Learn more about Grouparoo's Postgres plugin</a>
```
Comment (teallarson) [inline pages/integrations/postgres-to-salesforce.tsx]: Postgres integration?

File: pages/integrations/postgres-to-salesforce.tsx [hunk 6]
```diff
@@ -231,6 +232,79 @@ export default function PostgresToSalesforce() {
...
+              <p>
+                Salesforce is a leader in the SaaS-based CRM space. They have an
+                extensive product suite that encompasses marketing, sales,
+                support, and many other functions. Salesforce allows companies
+                and organizations to manage the customer experience and journey
+                across many different channels.
+              </p>
+              <small>
+                <Link href="/integrations/destinations/salesforce">
+                  <a>Learn more about Grouparoo's Salesforce plugin</a>
```
Comment (teallarson) [inline pages/integrations/postgres-to-salesforce.tsx]: Salesforce data integration?

File: pages/pricing.tsx [hunk 7]
```diff
@@ -465,6 +466,72 @@ export default function PricingPage() {
...
+        style={{
+          textAlign: "center",
+          backgroundImage: "linear-gradient(#242436,#43435F)",
+        }}
+        className="text-white homePageSection"
+      >
+        <Container>
+          <Row>
+            <Col>
+              <h2>Frequently Asked Questions</h2>
```
Comment (teallarson) [inline pages/pricing.tsx:482]: Awesome

**Review by teallarson — APPROVED** [review]
Great updates!  I think in addition to SEO, they add some useful clarity for people who visit these pages.  

A few notes on link anchor text and one on phrasing, but none are blocking.

File: pages/docs/support/permissions.mdx (hunk 1 above)
Comment (andyjih) [reply pages/docs/support/permissions.mdx:9]: Yes! Good call.

**Pull request merged**